lib/modules/manager/composer/utils.spec.ts

```diff
@@ -316,6 +317,74 @@ describe('modules/manager/composer/utils', () => {
     });
   });
 
+  describe('getComposerUpdateArguments', () => {
+    it('does not request an update with minimal changes with 2.6.0', () => {
```

Comment:
- use each to test multiple versions
- add tests for `>=2.6` and `>=3.0.0`

Reply:
>use each to test multiple versions

This would make this test inconsistent with all the others of this file. Are you fine with this?

Reply:
yes